Client/Frontend/Home/Wallpapers/v1/UI/WallpaperSelectorViewController.swift

```diff
@@ -58,14 +58,6 @@ class WallpaperSelectorViewController: WallpaperBaseViewController, Themeable {
         return collectionView
     }()
 
-    private lazy var settingsButton: ResizableButton = .build { button in
-        button.titleLabel?.font = DynamicFontHelper.defaultHelper.preferredFont(withTextStyle: .body,
-                                                                                size: 16)
-        button.titleLabel?.textAlignment = .center
-        button.setTitle(.Onboarding.WallpaperSelectorAction, for: .normal)
```

Comment:
Could you please remove the string from `Strings.swift` as well?

Reply:
yes, thank you for pointing this out!